### PR TITLE
CXX-705 banish whitespace from evergreen config

### PR DESCRIPTION
**Pull request by amidvidy**

### CODE REVIEW

File: .mci.yml [hunk 1]
```diff
@@ -9,38 +9,38 @@ cxx_driver_variables:
...
-    - name: "integration test 2.6"
-    - name: "integration test 3.0"
-    - name: "integration test latest"
-    - name: "client test 2.2"
-    - name: "client test 2.4"
-    - name: "client test 2.6"
-    - name: "client test 3.0"
-    - name: "client test latest"
+    - name: "unit-test"
+    - name: "integraion-test-2.2"
```
Comment (acmorrow) [inline .mci.yml]: have a typo here

File: .mci.yml [hunk 2]
```diff
@@ -367,7 +367,7 @@ tasks:
                 set -o verbose
                 ${scons|scons} ${scons_flags} ${warning_flags} ${compile_flags} ${toolchain_flags} ${extrapath} ${dllpath} ${cpppath} ${libpath} unit
 
-    - name: "integration test 2.2"
+    - name: "integraion-test-2.2"
```
Comment (acmorrow) [inline .mci.yml]: ditto

**Comment by acmorrow** [comment]
LGTM mod typos

File: .mci.yml (hunk 2 above)
Comment (amidvidy) [inline .mci.yml]: failed find and replace :'(

**Pull request merged**